### DISCUSSION TITLE
Extension bookmark (or favorites)?

### DISCUSSION THREAD

**Discussion by nothingness6** [microsoft/vscode-discussions#572] [Extension Development QnA] [answered]
Since I installed several AI coding helpers, they should be switched on/off due to the conflict, and it's kind of pain in the ass. I have to scroll down at the bottom of the extensions and turn it off every time I should do. 
I'd like to bookmark the ones what I use mostly and pick them up at a glance. Is there a function like that in VScode?

**Comment by alefragnani** [comment, marked as answer]
Hi @nothingness6 ,

Have you tried to use [Profiles in Visual Studio Code](https://code.visualstudio.com/docs/editor/profiles)? It seems to fit just fine for your use case. 

Install the _used mostly_ in the Default profile and leave the _other AI Coding Helpers_ to other profiles. So, when you need some specific AI coding helper, simply switch to its profile. 

Hope this helps

**Reply by nothingness6** [reply]
Hi, @alefragnani.
Thanks for the tip. So, I can switch profiles instead of extensions. Interesting. 
It helps. I'll give it a shot. 
Thanks again!

Regards,
Jean
